src/app/login/login.component.html

```diff
@@ -1,3 +1,125 @@
-<p>
-  login works!
-</p>
+
+<div class="section2" id="two" style="text-align: center;"> 
+   		
+  <br>
+  <h1 class="center">Sign-In With</h1>
+  <br>
+
+ <button class="bfacebook">
+<a class="btn btn-block btn-social btn-facebook center" style="width: 380px;">
+<span class="fa fa-facebook"></span> Sign in with Facebook
+</a></button>
+<br><br>
+
+<button class="bgoogle">
+<a class="btn btn-block btn-social btn-google center" style="width: 380px;">
+<span class="fa fa-google"></span> Sign in with Google
+</a></button>
+<br>  <h2 class="heading-2">OR</h2>
+<br>
+  <form>
+   
+  
+<div class="form-group">
+<label for="exampleInputEmail1">Email address</label>
+<input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="email@example.com">
+<small id="emailHelp" class="form-text text-muted">We'll never share your email with anyone else.</small>
+</div>
+<div class="form-group">
+<label for="exampleInputPassword1">Password</label>
+<input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
+</div>
+
+<div class="form-group">
+<div class="form-check">
+<input class="form-check-input" type="checkbox" id="gridCheck1">
+<label class="form-check-label" for="gridCheck">
+Remember Me
+</label>
+</div>
+</div>
+
+<a href="#" class="forget" style="text-align: center;">Forget Password?</a>
+<br><br>
+<button type="submit" class="btn btn-primary">Sign-In</button>
+<br>
+<a href="#" class="new">New Member?</a>
+
+<br><br>
+
+</form>
+
+</div>
+
+
+
+
+
+<div class="section1" id="one" style="text-align: center;">
+
+
+<br>
+<h1 class="center">Sign Up</h1>
+<br>
+<form>
+ <div class="form-group">
+<label for="exampleInputName">Name</label>
+<input type="name" class="form-control" id="exampleInputName" aria-describedby="nameHelp" placeholder="Enter your Name">
+</div>
+  <div class="form-group">
+<label for="exampleInputEmail1">Email address</label>
+<input type="email" class="form-control" id="exampleInputEmail" aria-describedby="emailHelp" placeholder="Enter email">
+<small id="emailHelp" class="form-text text-muted">We'll never share your email with anyone else.</small>
+</div>
+<div class="form-group">
+<label for="exampleSelect1">Gender</label>
+<select class="form-control" id="exampleSelect1">
+ <option>Male</option>
+ <option>Female</option>
+</select>
+</div>
+<label for="exampleInputFile">Date of Birth</label><br>
+<form action="/action_page.php">
```

Comment:
remove this link.
It is not a php project.

Reply:
> remove this link.
> It is not a php project.

Sir without this link calendar is not showing. I tried everything but it is not setting accordingly except this

Reply:
Use ngb-datepicker or material datepicker and remove this

Reply:
yes sir I have removed php link and date picker is working.